apps/api/src/user/user.e2e.spec.ts

```diff
@@ -41,7 +41,7 @@ describe('User Controller Tests', () => {
 
   beforeEach(async () => {
     adminUser = await userService.createUser({
-      email: 'admin@keyshade.xyz',
+      email: process.env.FEEDBACK_FORWARD_EMAIL,
```

Comment:
This would remain the same as before. It is purely for testing purpose and won't matter.

Reply:
Done.